pkg/jx/cmd/get_applications.go

```diff
@@ -114,33 +116,116 @@ func NewCmdGetApplications(f Factory, in terminal.FileReader, out terminal.FileW
 
 // Run implements this command
 func (o *GetApplicationsOptions) Run() error {
-	f := o.Factory
-	client, currentNs, err := f.CreateJXClient()
+	kubeClient, _, err := o.KubeClient()
 	if err != nil {
 		return err
 	}
-	kubeClient, _, err := o.KubeClient()
+
+	namespaces, envApps, envNames, apps, err := o.getAppData(kubeClient)
 	if err != nil {
 		return err
 	}
+
+	util.ReverseStrings(namespaces)
```

Comment:
If you have any idea why this is being reversed, a comment would be good. I can't remember whether I put it in or not, but I have no idea why it's being done here